StripeFinancialConnections/StripeFinancialConnections/Source/Common/FinancialConnectionsNavigationController.swift

```diff
@@ -169,20 +170,54 @@ extension FinancialConnectionsNavigationController {
                 stripeLogoImageView.tintColor = UIColor.textActionPrimary
                 stripeLogoImageView.contentMode = .scaleAspectFit
                 stripeLogoImageView.sizeToFit()
+
+                let height: CGFloat = 20
+                let stripeLogoImageViewWidth = stripeLogoImageView.bounds.width * (height / max(1, stripeLogoImageView.bounds.height))
                 stripeLogoImageView.frame = CGRect(
                     x: 0,
                     y: 0,
-                    width: stripeLogoImageView.bounds.width * (20 / max(1, stripeLogoImageView.bounds.height)),
-                    height: 20
+                    width: stripeLogoImageViewWidth,
+                    height: height
                 )
-                // If `titleView` is directly set to the `UIImageView`
+
+                let logoView: UIView
```

Comment:
I like how you separated this logic into two parts, named it logo view, and then kept the variable uninitialized where Swift compiler will ensure its set

Makes it clear to read IMO!